### PR TITLE
Add protobuf-d as a submodule to Agora

### PR DESCRIPTION
**Pull request by Geod24**
Part of #203

### CODE REVIEW

File: submodules/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ This data can also be gathered by running `git submodules`.
 | memutils         | 0.4.13       |              |          |
 | mir-linux-kernel | 1.0.1        |              |          |
 | openssl          | 1.1.6+1.0.1g |              |          |
+| protobuf-d       | a89aa09      |              |          |
```
Comment (AndrejMitrovic) [inline submodules/README.md:21]: Why not a released version?

Comment (AndrejMitrovic) [reply submodules/README.md:21]: I mean a tag.

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by Geod24** [comment]
Because there's no tagged version which include the commit which fixes 120 lines of deprecations. Namely [this](https://github.com/dcarp/protobuf-d/commit/686d60c137ccf64c876ef4e7760e41e60d49d333).

**Comment by Geod24** [comment]
Dockercloud will be green after merge

**Pull request merged**